.azure/OneBranch.Publish.yml

```diff
@@ -22,40 +22,7 @@ parameters:
 - name: kernel
   type: object
   default:
-  # build on ubuntu 20.04 openssl 1.1
-  - name: kernel5_4
-    type: object
-    default:
-    - name: RPM
-      destinations:
-        - microsoft-sles12-prod-yum     # 12 3.12
-        - microsoft-sles15-prod-yum     # 15 4.12
-        - microsoft-centos7-prod-yum    # 7 3.10
-        - microsoft-centos8-prod-yum    # 8 4.18
-        - microsoft-opensuse15-prod-yum # 15 4.12
-        - microsoft-fedora32-prod-yum   # 32 5.6
-        - microsoft-fedora33-prod-yum   # 33 5.8
-        - microsoft-fedora34-prod-yum   # 34 5.11
-        - microsoft-fedora35-prod-yum   # 35 5.14
-        - microsoft-rhel7.3-prod-yum    # 7.3 3.10
-        - microsoft-rhel8.0-prod-yum    # 8.0 4.18
-        - microsoft-rhel8.1-prod-yum    # 8.1 4.18
-    - name: DEB
-      destinations:
-        - microsoft-ubuntu-xenial-prod-apt   # 16.04 4.4
-        - microsoft-ubuntu-bionic-prod-apt   # 18.04 4.15
-        - microsoft-ubuntu-focal-prod-apt    # 20.04 5.4
-        - microsoft-ubuntu-groovy-prod-apt   # 20.10 5.8
-        - microsoft-ubuntu-hirsute-prod-apt  # 21.04 5.11
-        - microsoft-debian-stretch-prod-apt  #  9 4.9
-        - microsoft-debian-buster-prod-apt   # 10 4.19
-        - microsoft-debian-bullseye-prod-apt # 11 5.10
-    - name: CBL
-      destinations:
-        - cbl-mariner-1.0-prod-Microsoft-x86_64-rpms-yum
```

Comment:
no more mariner support?

Reply:
We need to figure out how to build for a more recent version. The old one used v1.1 openssl.